pennylane/ops/op_math/exp.py

```diff
@@ -24,7 +24,7 @@
     expand_matrix,
     OperatorPropertyUndefined,
 )
-from pennylane.pauli import is_pauli_word, pauli_word_to_string
```

Comment:
@albi3ro, I had to move this import down to avoid circular import errors. Not the biggest fan of this solution so let me know if you have any alternatives in mind !